### DISCUSSION TITLE
请问是否有定时消息的功能

### DISCUSSION THREAD

**Discussion by TianLuoRain** [mamoe/mirai#1186] [Q&A] [answered]
现有一个定时发送消息的需求，但翻找文档和讨论并没有找到相关内容，想问问Mirai是否提供定时消息的功能？如果没有的话恐怕就只能自己试着实现了。X﹏X

**Comment by Karlatemp** [comment, marked as answer]
并没有，定时任务并不属于 `mirai-core` 的任务, 你可以自行搜索定时任务库

**Reply by TianLuoRain** [reply]
谢谢，我了解了。
——————
回帖好快啊，我刚把网页关了邮件就响了。🤣🤣
